File: benchmarks/startup/ref.c
/* startup reference, C. */
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char **argv) {
  long long n = argc > 1 ? atoll(argv[1]) : 0;
  printf("startup %lld\n", n * n + 1);
  return 0;
}

File: benchmarks/startup/ref.go
// startup reference, Go.
package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {
	var n int64
	if len(os.Args) > 1 {
		n, _ = strconv.ParseInt(os.Args[1], 10, 64)
	}
	fmt.Printf("startup %d\n", n*n+1)
}
